site/kubernetes/operator/using-operator.md

```diff
@@ -1034,3 +1037,23 @@ or use
 <pre class="lang-bash">
 kubectl delete -f INSTANCE.yaml
 </pre>
+
+## <a id='pause' class='anchor' href='#pause'>Pause Reconciliation for a RabbitMQCluster</a>
+
+If you wish to pause reconciliation for a RabbitMQ instance to stop the cluster operator updating and watching the instance. You can set a special label on your RabbitmqCluster.
+
+This feature can be used if you wish to upgrade to a new version of the cluster operator but do not wish for the operator to start updating some of your RabbitmqCluster. Please be aware that pausing reconciliation means that the operator will not watch this RabbitmqCluster until the special label is removed. Any updates to the paused RabbitmqCluster will be ignored by the operator and if you accidentally deletes a child resource of the RabbitmqCluster (e.g. the Stateful Set or Service object), deleted object won't be recreated automatically. We do not recommend using this feature unless absolutely necessary.
+
+To pause reconciliation, you can set the special label like
+
+<pre class="lang-bash">
+kubectl label rabbitmqclusters INSTANCE-NAME rabbitmq.com/pauseReconciliation=true
+</pre>
+
+where `INSTANCE` is the name of your RabbitmqCluster,
+
+To resume reconciliation, you can simply remove the label like
```

Comment:
`like` -> `by running` (or `like this:`)

Reply:
👍